src/ContractEditor/index.js

```diff
@@ -49,24 +52,30 @@ const contractProps = {
 /**
  * A rich text contract editor
  */
-const ContractAssembler = props => (<div><MarkdownEditor markdown={props.markdown}
-        onChange={props.onChange}
-        lockText={true}
-        plugins={props.plugins}/></div>);
-
-const ContractEditor = () => ContractAssembler(contractProps);
+const ContractEditor = props => (
+  <MarkdownEditor
+    markdown={props.markdown || contractProps.markdown}
```

Comment:
Where is `contractProps` coming from?

Reply:
it's defined earlier in that same file (line 46)